content/2_Bronze/Intro_Complete.mdx

```diff
@@ -167,6 +167,47 @@ Rounding suffices (`(int)round(pow(max_dist,2))`), but the takeaway is that you
 should stick with integers whenever possible.
 
 </CPPSection>
+<PySection>
+
+The following solution correctly stores the maximum distance in a floating point
+variable.
+
+<Warning>
+
+Make sure run using PyPy 3. Program will run out of time and/or memory unless you do.
+
+</Warning>
+The following solution correctly stores the maximum distance in a floating point
+variable.
+```py
+import math
```

Comment:
use tabs instead of spaces to indent

Reply:
How?

Reply:
an easy way is to use [Editor](https://usaco.guide/editor/); you can paste your code, then right click > command palette > convert indentation to tabs